pages/projects.tsx

```diff
@@ -27,7 +27,7 @@ const projects = [
       description: "A website built to simplify the experience of searching courses and building timetables for the University of Victoria (UVic).",
       status: "Active",
       tech: ["TypeScript", "React", "Firebase"],
-      meetingTime: "Fridays at 6-7pm",
+      meetingTime: "Waiting for updates",
```

Comment:
@liamshatzel do you have an update for this?

Reply:
Also, should Scholar-U have their meetings as "Undecided" or "No Meetings Planned". @liamshatzel you were talking to them right?

Reply:
Confirmed in meeting, Courseup should be "Undecided" for now.

Scholar-U/Scholarship Finder should be "No Meetings Planned"